### PR TITLE
Add About page

### PR DESCRIPTION
**Pull request by ZitaNemeckova**
Screenshot 2024-10-13 at 9 42 36

Plus a small change to README for Docker users

### CODE REVIEW

File: src/containers/about-project.tsx [hunk 1]
```diff
@@ -14,8 +15,47 @@ class AboutProject extends Component<RouteProps> {
...
+                    href={'https://github.com/pulp/pulp_ui'}
+                  >
+                    {'GitHub repository'}
+                  </ExternalLink></CardFooter>
+                </Card>
+              <Card ouiaId="BasicCard">
+                <CardTitle>Issue tracker</CardTitle>
+                <CardBody>{`If you find a bug or have an idea for enhancement, please feel free to file a Github issue. Thank you for contributing to improvement of the project.`}</CardBody>
+                <CardFooter> <ExternalLink
+                  href={'https://github.com/pulp/pulp_ui/issues'}
```
Comment (gerrod3) [inline src/containers/about-project.tsx]:
```suggestion
                  href={'https://github.com/pulp/pulp-ui/issues'}
```

File: src/containers/about-project.tsx [hunk 2]
```diff
@@ -14,8 +15,47 @@ class AboutProject extends Component<RouteProps> {
...
+              <CardBody style={{'text-align': 'center'}}>Pulp is an open source project that makes it easy for developers to fetch, upload, and distribute Software Packages on-prem or in the cloud.</CardBody>
+            </Card>
+          <Card ouiaId="BasicCard">
+            <CardBody>
+              <Gallery hasGutter style={{'grid-template-columns': '32.5% 32.5% 32.5%'}}>
+                <Card ouiaId="BasicCard">
+                  <CardTitle>Pulp UI</CardTitle>
+                  <CardBody>{t`This project is an attempt to up-cycle Ansible Galaxy UI codebase. The first version contains mostly saved code with minor fixes. In the future more types of content will be added. We welcome kindly contributions.`}</CardBody>
+                  <CardFooter> <ExternalLink
+                    href={'https://github.com/pulp/pulp_ui'}
```
Comment (gerrod3) [inline src/containers/about-project.tsx]:
```suggestion
                    href={'https://github.com/pulp/pulp-ui'}
```

File: src/containers/about-project.tsx [hunk 3]
```diff
@@ -14,8 +15,47 @@ class AboutProject extends Component<RouteProps> {
       <>
         <BaseHeader title={t`About project`} />
         <Main>
-          <Gallery hasGutter></Gallery>
-          <br />
+            <Card ouiaId="BasicCard">
+              <CardTitle>
+              <Brand style={{'display': 'block', 'margin-left': 'auto', 'margin-right': 'auto'}} src={PulpLogo} alt="PatternFly logo"/>
```
Comment (gerrod3) [inline src/containers/about-project.tsx]:
```suggestion
              <Brand style={{'display': 'block', 'margin-left': 'auto', 'margin-right': 'auto'}} src={PulpLogo} alt="Pulp logo"/>
```

File: src/containers/about-project.tsx [hunk 4]
```diff
@@ -14,8 +15,47 @@ class AboutProject extends Component<RouteProps> {
...
+              </CardTitle>
+              <CardBody style={{'text-align': 'center'}}>Pulp is an open source project that makes it easy for developers to fetch, upload, and distribute Software Packages on-prem or in the cloud.</CardBody>
+            </Card>
+          <Card ouiaId="BasicCard">
+            <CardBody>
+              <Gallery hasGutter style={{'grid-template-columns': '32.5% 32.5% 32.5%'}}>
+                <Card ouiaId="BasicCard">
+                  <CardTitle>Pulp UI</CardTitle>
+                  <CardBody>{t`This project is an attempt to up-cycle Ansible Galaxy UI codebase. The first version contains mostly saved code with minor fixes. In the future more types of content will be added. We welcome kindly contributions.`}</CardBody>
```
Comment (gerrod3) [inline src/containers/about-project.tsx]:
```suggestion
                  <CardBody>{t`This project is an attempt to up-cycle the Ansible Galaxy UI codebase. The first version contains mostly saved code with minor fixes. In the future more types of content will be added. We welcome kindly any contributions.`}</CardBody>
```

**Review by gerrod3 — CHANGES_REQUESTED** [review]

**Comment by himdel** [comment]
(~~needs a rebase and prettier~~, but LGTM :))
.. and l10n :)

File: src/containers/about-project.tsx [hunk 5]
```diff
@@ -14,8 +22,69 @@ class AboutProject extends Component<RouteProps> {
...
+                hasGutter
+                style={{ 'grid-template-columns': '32.5% 32.5% 32.5%' }}
+              >
+                <Card ouiaId='BasicCard'>
+                  <CardTitle>Pulp UI</CardTitle>
+                  <CardBody>{t`This project is an attempt to up-cycle the Ansible Galaxy UI codebase. The first version contains mostly saved code with minor fixes. In the future more types of content will be added. We welcome kindly any contributions.`}</CardBody>
+                  <CardFooter>
+                    {' '}
+                    <ExternalLink href={'https://github.com/pulp/pulp-ui'}>
+                      {'GitHub repository'}
```
Comment (himdel) [inline src/containers/about-project.tsx]:
```suggestion
                      {t`GitHub repository`}
```

File: src/containers/about-project.tsx [hunk 6]
```diff
@@ -14,8 +22,69 @@ class AboutProject extends Component<RouteProps> {
...
+          <Card ouiaId='BasicCard'>
+            <CardTitle>
+              <Brand
+                style={{
+                  display: 'block',
+                  'margin-left': 'auto',
+                  'margin-right': 'auto',
+                }}
+                src={PulpLogo}
+                alt='Pulp logo'
```
Comment (himdel) [inline src/containers/about-project.tsx]:
```suggestion
                alt={t`Pulp logo`}
```

File: src/containers/about-project.tsx [hunk 7]
```diff
@@ -14,8 +22,69 @@ class AboutProject extends Component<RouteProps> {
...
+            </CardBody>
+          </Card>
+          <Card ouiaId='BasicCard'>
+            <CardBody>
+              <Gallery
+                hasGutter
+                style={{ 'grid-template-columns': '32.5% 32.5% 32.5%' }}
+              >
+                <Card ouiaId='BasicCard'>
+                  <CardTitle>Pulp UI</CardTitle>
```
Comment (himdel) [inline src/containers/about-project.tsx]:
```suggestion
                  <CardTitle>{APPLICATION_NAME}</CardTitle>
```

File: src/containers/about-project.tsx [hunk 8]
```diff
@@ -14,8 +22,69 @@ class AboutProject extends Component<RouteProps> {
...
+                  <CardTitle>Get involved</CardTitle>
+                  <CardBody>{`Join our communication channels and get to know the contributors and users of Pulp's strong ecosystem.`}</CardBody>
+                  <CardFooter>
+                    {' '}
+                    <ExternalLink
+                      href={
+                        'https://pulpproject.org/help/community/get-involved/'
+                      }
+                    >
+                      {'Community'}
```
Comment (himdel) [inline src/containers/about-project.tsx]:
```suggestion
                      {t`Community`}
```

File: src/containers/about-project.tsx [hunk 9]
```diff
@@ -14,8 +22,69 @@ class AboutProject extends Component<RouteProps> {
...
+                    {' '}
+                    <ExternalLink
+                      href={'https://github.com/pulp/pulp-ui/issues'}
+                    >
+                      {'GitHub issues'}
+                    </ExternalLink>
+                  </CardFooter>
+                </Card>
+                <Card ouiaId='BasicCard'>
+                  <CardTitle>Get involved</CardTitle>
```
Comment (himdel) [inline src/containers/about-project.tsx]:
```suggestion
                  <CardTitle>{t`Get involved`}</CardTitle>
```

File: src/containers/about-project.tsx [hunk 10]
```diff
@@ -14,8 +22,69 @@ class AboutProject extends Component<RouteProps> {
...
+                  <CardBody>{t`This project is an attempt to up-cycle the Ansible Galaxy UI codebase. The first version contains mostly saved code with minor fixes. In the future more types of content will be added. We welcome kindly any contributions.`}</CardBody>
+                  <CardFooter>
+                    {' '}
+                    <ExternalLink href={'https://github.com/pulp/pulp-ui'}>
+                      {'GitHub repository'}
+                    </ExternalLink>
+                  </CardFooter>
+                </Card>
+                <Card ouiaId='BasicCard'>
+                  <CardTitle>Issue tracker</CardTitle>
```
Comment (himdel) [inline src/containers/about-project.tsx]:
```suggestion
                  <CardTitle>{t`Issue tracker`}</CardTitle>
```

File: src/containers/about-project.tsx [hunk 11]
```diff
@@ -14,8 +22,69 @@ class AboutProject extends Component<RouteProps> {
...
+                  <CardBody>{`If you find a bug or have an idea for enhancement, please feel free to file a Github issue. Thank you for contributing to improvement of the project.`}</CardBody>
+                  <CardFooter>
+                    {' '}
+                    <ExternalLink
+                      href={'https://github.com/pulp/pulp-ui/issues'}
+                    >
+                      {'GitHub issues'}
```
Comment (himdel) [inline src/containers/about-project.tsx]:
```suggestion
                      {t`GitHub issues`}
```

File: src/containers/about-project.tsx [hunk 12]
```diff
@@ -14,8 +22,69 @@ class AboutProject extends Component<RouteProps> {
...
+                  'margin-right': 'auto',
+                }}
+                src={PulpLogo}
+                alt='Pulp logo'
+              />
+            </CardTitle>
+            <CardBody style={{ 'text-align': 'center' }}>
+              Pulp is an open source project that makes it easy for developers
+              to fetch, upload, and distribute Software Packages on-prem or in
+              the cloud.
```
Comment (himdel) [inline src/containers/about-project.tsx]:
```suggestion
              {t`Pulp is an open source project that makes it easy for developers to fetch, upload, and distribute Software Packages on-prem or in the cloud.`}
```

File: src/containers/about-project.tsx [hunk 13]
```diff
@@ -14,8 +22,69 @@ class AboutProject extends Component<RouteProps> {
       <>
         <BaseHeader title={t`About project`} />
         <Main>
-          <Gallery hasGutter></Gallery>
-          <br />
+          <Card ouiaId='BasicCard'>
```
Comment (himdel) [inline src/containers/about-project.tsx]:
```suggestion
          <Card>
```

File: src/containers/about-project.tsx [hunk 14]
```diff
@@ -14,8 +22,69 @@ class AboutProject extends Component<RouteProps> {
...
+                alt='Pulp logo'
+              />
+            </CardTitle>
+            <CardBody style={{ 'text-align': 'center' }}>
+              Pulp is an open source project that makes it easy for developers
+              to fetch, upload, and distribute Software Packages on-prem or in
+              the cloud.
+            </CardBody>
+          </Card>
+          <Card ouiaId='BasicCard'>
```
Comment (himdel) [inline src/containers/about-project.tsx]:
```suggestion
          <Card>
```

File: src/containers/about-project.tsx [hunk 15]
```diff
@@ -14,8 +22,69 @@ class AboutProject extends Component<RouteProps> {
...
+                  <CardTitle>Pulp UI</CardTitle>
+                  <CardBody>{t`This project is an attempt to up-cycle the Ansible Galaxy UI codebase. The first version contains mostly saved code with minor fixes. In the future more types of content will be added. We welcome kindly any contributions.`}</CardBody>
+                  <CardFooter>
+                    {' '}
+                    <ExternalLink href={'https://github.com/pulp/pulp-ui'}>
+                      {'GitHub repository'}
+                    </ExternalLink>
+                  </CardFooter>
+                </Card>
+                <Card ouiaId='BasicCard'>
```
Comment (himdel) [inline src/containers/about-project.tsx]:
```suggestion
                <Card>
```

File: src/containers/about-project.tsx [hunk 16]
```diff
@@ -14,8 +22,69 @@ class AboutProject extends Component<RouteProps> {
...
+                  <CardFooter>
+                    {' '}
+                    <ExternalLink
+                      href={'https://github.com/pulp/pulp-ui/issues'}
+                    >
+                      {'GitHub issues'}
+                    </ExternalLink>
+                  </CardFooter>
+                </Card>
+                <Card ouiaId='BasicCard'>
```
Comment (himdel) [inline src/containers/about-project.tsx]:
```suggestion
                <Card>
```

File: src/containers/about-project.tsx [hunk 17]
```diff
@@ -14,8 +22,69 @@ class AboutProject extends Component<RouteProps> {
...
+              the cloud.
+            </CardBody>
+          </Card>
+          <Card ouiaId='BasicCard'>
+            <CardBody>
+              <Gallery
+                hasGutter
+                style={{ 'grid-template-columns': '32.5% 32.5% 32.5%' }}
+              >
+                <Card ouiaId='BasicCard'>
```
Comment (himdel) [inline src/containers/about-project.tsx]:
```suggestion
                <Card>
```

**Pull request merged**